app/views/user/_training_modules.html.slim

```diff
@@ -0,0 +1,7 @@
+h3 Training Modules
+ul 
+  - TrainingModule.all.each do |t|
+    li=t.title
+    = @user.started?(training_module: t) ? link_to("Resume", url_for(@user.last_page_for(training_module: t))) : "Not started"
```

Comment:
This would get flagged for the link being outside the list item.

Reply:
OK, I've removed the list for now - this is just until a proper design comes through